scripts/mkstdlibs.py

```diff
@@ -4,7 +4,7 @@
 
 URL = "https://docs.python.org/{}/objects.inv"
 PATH = "isort/stdlibs/py{}.py"
-VERSIONS = [("2", "7"), ("3", "5"), ("3", "6"), ("3", "7"), ("3", "8"), ("3", "9"), ("3", "10")]
+VERSIONS = [("2", "7"), ("3", "6"), ("3", "7"), ("3", "8"), ("3", "9"), ("3", "10"), ("3", "11")]
```

Comment:
`isort/stdlibs/__init__.py` and `isort/stdlibs/py3.py` still need to be updated.